internal_ws/ykpack/src/types.rs

```diff
@@ -492,6 +492,10 @@ pub enum Statement {
     /// Cast a value into another. Since the cast type and the destination type are the same, we
     /// only need the latter.
     Cast(IRPlace, IRPlace),
+    /// Marks the spot where we want trace stitching to start.
```

Comment:
Have we defined "stitching" anywhere? Should we just say explicitly what this does instead?

Something like:
> The position we jump to when we encounter a `LoopEnd` bytecode. There must be only one `LoopStart` bytecode per trace.

Similarly for `LoopEnd`.

Reply:
I'm actually not sure I even like the term "stitching", which to me suggests putting multiple things together, while this is about adding a loop.

Reply:
I could (in real life, using a needle) stitch a straight line of fabric with a loop half way up it's length, and that'd be quite a lot like out trace code. So the term doesn't bother me, but it should at the least be defined.

Reply:
I was thinking the same as @ltratt that stitching implies to me stitching together multiple _different_ traces. I've just used it in lack of a better alternative.

Reply:
I never thought of it from that POV, but we can still consider other options.

How about "trace folding", as it implies a transform on a single entity?

Reply:
I could just change the doc string to "Mark the spot where we want the trace to loop [on itself]".

Reply:
> I could just change the doc string to "Mark the spot where we want the trace to loop [on itself]".

That seems better than "stitching" to me.

Reply:
Works for me too.

Reply:
Fixed in 655129f.